node/config/def.go

```diff
@@ -109,6 +109,7 @@ func DefaultFullNode() *FullNode {
 		Cluster: *DefaultUserRaftConfig(),
 		Fevm: FevmConfig{
 			EnableEthRPC:                 false,
+			EnableActorEventsAPI:         false,
```

Comment:
It seems weird for this to live under FEVM config

Reply:
Events is a functionality offered by the FVM that built-in Actor uses. Do we differentiate between the `FVM` and the `FEVM` in Lotus ?